packages/vue-renderer/src/renderers/spa.js

```diff
@@ -13,27 +12,21 @@ export default class SPARenderer extends BaseRenderer {
 
     this.cache = new LRU()
 
-    // Add VueMeta to Vue (this is only for SPA mode)
-    // See app/index.js
-    Vue.use(VueMeta, {
+    this.vueMetaConfig = {
       keyName: 'head',
       attribute: 'data-n-head',
       ssrAttribute: 'data-n-head-ssr',
+      ssrAppId: '1',
       tagIDKeyName: 'hid'
-    })
+    }
   }
 
   createRenderer () {
     return createRenderer()
   }
 
-  async getMeta () {
-    const vm = new Vue({
-      render: h => h(), // Render empty html tag
-      head: this.options.head || {}
-    })
-    await this.vueRenderer.renderToString(vm)
-    return vm.$meta().inject()
+  getMeta () {
+    return VueMeta.generate(this.options.head || {}, this.vueMetaConfig)
```

Comment:
What do you think renaming `getMeta` to `generateMeta` for consistency?

Reply:
I thought about just removing that function but wasnt sure if that would be a breaking change. Not sure if its programmatically possible to extend SPARenderer and register your own. If not them I am happy to change it into whatever you like :)

Reply:
Unless if we had docs or typescript which I don't think we have. That's an internal breaking change. /cc @clarkdo are you up to this change?

Reply:
Have removed the getMeta function (doesnt seem to be possible to override the SPArenderer as the renderers are hardcoded loaded in renderer.js)